src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,9 @@
+.logo {
+  &__icon {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+// usually logo(with name of company) need to be used in header and footer both
+//so I decided to separate that in special block
```

Comment:
On my opinion we dont need to create separate block for icon. Because icon is a single element of block.